tests/components/zha/test_button.py

```diff
@@ -48,6 +52,35 @@ async def contact_sensor(hass, zigpy_device_mock, zha_device_joined_restored):
     return zha_device, zigpy_device.endpoints[1].identify
 
 
+@pytest.fixture
+async def tuya_water_valve(hass, zigpy_device_mock, zha_device_joined_restored):
+    """Tuya Water Valve fixture."""
+
+    class TuyaManufCluster(ManufacturerSpecificCluster):
```

Comment:
you need to extend CustomCluster or you will interfere with Zigby's cluster registry. See here for an example: https://github.com/zigpy/zha-device-handlers/blob/c7f2c61fb71b0a4b01e17779644270ef9fc11326/zhaquirks/__init__.py#L178